packages/video_player/tizen/src/video_player.cc

```diff
@@ -455,14 +495,15 @@ void VideoPlayer::onErrorOccurred(int code, void *data) {
   LOG_DEBUG("[VideoPlayer.onErrorOccurred] error code: %s",
             get_error_message(code));
 
-  if (player->eventSink_) {
+  if (player->event_sink_) {
     LOG_INFO("[VideoPlayer.onErrorOccurred] send error event");
-    player->eventSink_->Error("Video player had error",
-                              get_error_message(code));
+    player->event_sink_->Error("Video player had error",
+                               get_error_message(code));
   }
 }
 
 void VideoPlayer::onVideoFrameDecoded(media_packet_h packet, void *data) {
+  LOG_DEBUG("onVideoFrameDecoded");
```

Comment:
IMO, This generates too many logs. Could you remove this line?

Reply:
ok, I delete it.